### PR TITLE
Spublock config

### PR DESCRIPTION
**Pull request by Zer0xFF**

### CODE REVIEW

File: Source/PS2VM.h [hunk 1]
```diff
@@ -163,6 +164,7 @@ class CPS2VM : public CVirtualMachine
 
 	int16 m_samples[BLOCK_SIZE * BLOCK_COUNT];
 	int m_currentSpuBlock = 0;
+	int m_SpuBlockCount = BLOCK_COUNT;
```
Comment (jpd002) [inline Source/PS2VM.h]: This should be ‘m_spuBlockCount’ to match the casing of other variables.

**Comment by jpd002** [comment]
This is pretty cool!

The setting should be managed by ‘CPS2VM’ to avoid setting the block count everytime we init the SPU handler. In other words, ‘m_spuBlockCount’ should be initialized in ‘CPS2VM’ ctor.

File: Source/ui_qt/Qt_ui/settingsdialog.ui [hunk 2]
```diff
@@ -260,6 +260,41 @@
...
+       <property name="geometry">
+        <rect>
+         <x>20</x>
+         <y>50</y>
+         <width>171</width>
+         <height>16</height>
+        </rect>
+       </property>
+       <property name="text">
+        <string>SPU Block Count*:</string>
```
Comment (jpd002) [inline Source/ui_qt/Qt_ui/settingsdialog.ui]: Why is there an asterisk here? Is it a typo?

Comment (Zer0xFF) [reply Source/ui_qt/Qt_ui/settingsdialog.ui]: I was going to add an explanation to it, but I wasn't sure what to write.

If you have a suggestion, else I'll remove it

**Comment by jpd002** [comment]
Tested it myself, it's pretty cool.

Instead of calling the setting "SPU Block Count", I think it might be better to call it "Buffer Size" (only in the UI, no need to rename the other stuff). I think it might be a good occasion to give a sensible default to that setting too, something like 100 seems to be good.

**Comment by Zer0xFF** [comment]
ya that "Buffer Size" probably works better, without much explanation.
Changed reduced default to 100 as well.

**Comment by jpd002** [comment]
Thanks a bunch!

**Comment by Zer0xFF** [comment]
👍 sure thing :)

**Pull request merged**